### ISSUE TITLE
Saints Row The Third: The Full Package [BLUS31062] - cellDiscGameGetBootDiscInfo

### ISSUE THREAD

**Issue by annahana** [RPCS3/rpcs3#1503]
RPCS3 v0.0.0.6-5-787619c
![rpcs3](https://cloud.githubusercontent.com/assets/13276505/13047546/28e6ce9a-d3e0-11e5-8c6b-412237ff9505.jpg)

{PPU Thread[0x1](main_thread)[0x0096a8ec]} sys_mmapper: sys_mmapper_allocate_address(size=0x40000000, flags=0x200, alignment=0x10000000, alloc_addr=*0x28d0df8)
{PPU Thread[0x1](main_thread)[0x00e657ac]} HLE: Last function: cellDiscGameGetBootDiscInfo (0xb72bc4e6)
{PPU Thread[0x1](main_thread)[0x00e657ac]} Exception: 
(in file C:\rpcs3\rpcs3\Emu\SysCalls\Modules\cellGame.cpp:643, in function cellDiscGameGetBootDiscInfo)
Please report this to the developers.

change .jpg to .log

**Comment by tambry** [comment]
I don't think it's really useful to keep track of unimplemented HLE functions as "issues". If anything, just keep them a single issue for the whole module. Otherwise you could very well make hundreds (if not thousands) of issues corresponding to every unimplemented HLE function.

**Comment by MarioSonic2987** [comment]
With #2390, outputs:
```
·F {PPU[0x1000000] Thread (main_thread) [0x00c90d04]} struct vm::access_violation thrown: Access violation writing address 0x0
```
https://gist.github.com/MarioSonic2987/6f80d89795d76e77a7dec9c59e6fa9a1

**Comment by B1ackDaemon** [comment]
@MarioSonic2987 game contains .sdat files, try to decrypt them.

**Comment by MarioSonic2987** [comment]
They're already decrypted. @B1ackDaemon

**Comment by AniLeo** [comment]
cellDiscGameGetBootDiscInfo got implemented on #2390 so that isn't an issue anymore. Game might hang on other unrelated functions though.
